src/shared_gui_components/BCLMeasureDialog.cpp

```diff
@@ -216,7 +216,8 @@ boost::optional<openstudio::BCLMeasure> BCLMeasureDialog::createMeasure() {
       result->changeUID();
 
       // change the files on disk
-      result->updateMeasureScript(m_bclMeasureToCopy->measureType(), measureType, m_bclMeasureToCopy->className(), className, name, description,
+      result->updateMeasureScript(m_bclMeasureToCopy->measureType(), measureType, m_bclMeasureToCopy->measureLanguage(),
```

Comment:
Is NREL going to post documentation on how to use Python measures?  Can you mix Ruby and Python measures in a workflow?

Reply:
Ultimately you will be able to mix them. This is experimental and for demonstration purposes right now (mixing measure types works on Mac and Windows, fails on Ubuntu, at the moment).
Aside from that they work exactly the same, syntax aside. Same layout, same methods. And the BCL now has a template for python measures too. 

Try this out in Ruby if you want to test it:

```
OpenStudio::BCLMeasure.new('A Test Python Measure', 'ATestPythonMeasure', 'atestpythonmeasure', 'enveloppe.fenestration', 'ModelMeasure'.to_MeasureType, 'Testing out the Python Measure', 'This is a measure that is written in Python', 'Python'.to_MeasureLanguage)
```

The installers also ship with new OSW in Examples/